src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/IntegrationTestSupportController.java

```diff
@@ -75,6 +85,27 @@ public void linkDefendantToClaim(
         ccdEventProducer.createCCDLinkDefendantEvent(claimReferenceNumber, defendantId);
     }
 
+    @PutMapping("/claims/{claimReferenceNumber}/defendant/{defendantUsername}/{defendantPassword}")
+    @ApiOperation("Link a claim to a defendant")
+    public void linkDefendantToClaim(
+        @PathVariable("claimReferenceNumber") String claimReferenceNumber,
+        @PathVariable("defendantUsername") String defendantUsername,
+        @PathVariable("defendantPassword") String defendantPassword
+    ) {
+        logger.info("Linking claim to defendant");
+            Claim claim = getClaim(claimReferenceNumber, null);
+        try {
```

Comment:
please take away try catch, are we are just catching to log the message, technically you see that in app insight.